libr/include/r_cmd.h

```diff
@@ -15,8 +15,15 @@ extern "C" {
 #define MACRO_LABELS 20
 #define R_CMD_MAXLEN 4096
 
-#define r_cmd_callback(x) int (*x)(void *data, const char *input)
-#define r_cmd_nullcallback(x) int (*x)(void *data)
+typedef enum r_cmd_status_t {
+	R_CMD_STATUS_OK = 0,
+	R_CMD_STATUS_INVALID,
+	R_CMD_STATUS_EXIT
+} RCmdStatus;
```

Comment:
How does RCmdStatus differ from RCoreCmdStatus? Looking at current code it seems that RCoreCmdStatus is result of evaluating shell expression but RCmdStatus is result of command callback. Are those really different things?

Reply:
no they aren't, but I want to remove RCoreCmdStatus and just bring it in r_cmd.h because I think it belongs there. Changing this here, however, would have required a lot of changes in cmd.c. Do you want me to do it here or in separate PR?

Reply:
If there are plans to unify then it's fine. Just wanted to clarify it.

Reply:
Yeah, I will probably follow up with a small PR that does a bit of a "cleaning".